spec/spec.md

```diff
@@ -134,21 +134,21 @@ sequenceDiagram
   rp -->> siop: Acknowledgement
 ```
 
-### Requirements Overview
+### Overview of the Open Standards Requirements
 
 - VCs MUST adhere to the [[ref: VC Data Model]] and be encoded as JSON and signed as JWT as defined in 6.3.1 of [[ref: VC Data Model]]. VCs encoded as JSON-LD and signed using Linked Data Proofs are NOT supported.
 - For key management and authentication, Self-Issued OpenID Connect Provider v2, an extension to OpenID Connect, MUST be used as defined in [[ref: SIOPv2]].
-- For transportation of VCs, OpenID Connect for Verifiable Presentations MUST be used as defined in [[ref: OIDC4VP]].
-- As the query language, Presentation Exchange MUST be used and conform to the syntax defined in [[ref: OIDC4VP]], a profile of [[ref: Presentation Exchange]].
-- Decentralized Identifiers (DIDs), as defined in [[ref: DID Core]], MUST be used as identifiers of the entities. Implementations MUST support did:ion as a mandatory DID method.
+- For transportation of VCs, OpenID for Verifiable Presentations MUST be used as defined in [[ref: OpenID4VP]].
+- As the query language, Presentation Exchange MUST be used and conform to the syntax defined in [[ref: OpenID4VP]], a profile of [[ref: Presentation Exchange]].
+- Decentralized Identifiers (DIDs), as defined in [[ref: DID Core]], MUST be used as identifiers of the entities. Implementations MUST support did:web and did:ion as a mandatory DID method as defined in [[ref: did-web]] and [[ref: did-ion]].
```

Comment:
first reference to did:web i believe. We may want to pull that piece out into its own PR to get consensus and add more elaboration to did:web as needed

Reply:
sounds good!